sktime/classification/kernel_based/_arsenal.py

```diff
@@ -174,8 +174,13 @@ def _fit(self, X, y):
         Changes state by creating a fitted model that updates attributes
         ending in "_" and sets is_fitted flag to True.
         """
+        if len(X.shape) == 2:
+            raise ValueError(
+                f"X is of type{type(X)} base classifier tags ="
+                f" {BaseClassifier._tags}  Arsenal get tags  ="
+                f" {BaseClassifier.get_tags(self)} Tags for Arsenal = {self._tags}"
+            )
```

Comment:
Why is this in Arsenal and no other class?

Reply:
Legacy of debugging :)